azerothcore/azerothcore-wotlk
# Core/Quest/Objects/Creatures Forge Camp Annihilated

Fixed Quest Forge Camp Annihilated: The Canons now shoot the camps







##### CHANGES PROPOSED:

- Updated SAI Script
- Updated Creatures ( Problem with LOS Targeting ) - Moved to Clear positions where can be targeted.
- Creature 68741  ( Fel Cannon - Fear Target )
- Creature 68742  ( Fel Cannon - Hate Target )

###### ISSUES ADDRESSED:


Closes 
https://github.com/azerothcore/azerothcore-wotlk/issues/1077

##### TESTS PERFORMED:

Builded without Errors, tested on two cores, everything is fine and working just as like blizzlike.
The Quest can be done without problem & Cannons now shoot the camps.

##### HOW TO TEST THE CHANGES:

1. .quest add 10011
2. .additem 25770
3. .go creature 68367
4. Use the Item from your backpack ( Fel Cannon Activator )
5. .additem 25771
6. .go creature 68740
7. Use the Item from your backpack ( Use Fel Cannon Activator )
8. The both should work and shoot the camps


##### KNOWN ISSUES AND TODO LIST:




##### Target branch(es):

Master
